app/src/main/java/org/fossasia/openevent/app/core/main/DrawerNavigator.java

```diff
@@ -19,6 +20,7 @@
     private final FragmentNavigator fragmentNavigator;
 
     private AlertDialog logoutDialog;
+    private final String GOOGLE_FORM_LINK = "https://docs.google.com/forms/d/e/1FAIpQLSfJ-v1mbmNp1ChpsikHDx6HZ5G9Bq8ELCivckPPcYlOAFOy2Q/viewform?usp=sf_link";
```

Comment:
You could probably add it in the Constants class where we store all our constants.

Reply:
I added it here because its being used in this class only. So it would be better to declare it here rather than using a Constants class for it.

Reply:
Ok, works in both ways.

Reply:
Yes, even constants should be used as locally as possible. We should not pollute global namespace